packages/universal-components/src/RangeDatePicker/RangeDatePickerContent.js

```diff
@@ -32,7 +34,12 @@ export default class RangeDatePickerContent extends React.Component<
   constructor(props: Props) {
     super(props);
     this.state = {
-      nextMonths: getMonths(props.numberOfRenderedMonths, props.weekStartsOn),
+      nextMonths: getMonths(
```

Comment:
It takes too many parameters, what about refactor to object input type? But nothing big deal just an idea.

Reply:
Done